build-system/tasks/helpers.js

```diff
@@ -404,6 +404,17 @@ function finishBundle(srcFilename, destDir, destFilename, options) {
   }
 }
 
+/**
+ * Returns array of relative paths to "dependencies" defined in package.json.
```

Comment:
nit: Should this comment say `"devDependencies"`?

Reply:
Oops, thanks.